### ISSUE TITLE
Indent mode incorrectly removes closing paren in presence of closing paren char

### ISSUE THREAD

**Issue by cursive-ghost** [parinfer/parinfer.js#106]
To reproduce, copy the following text to the reference implementation in paren mode:

```
(defn close-paren-and-newline [editor caret context]
  (close-and-newline editor \)))
```

Switch to Indent mode, and the last closing paren is removed. Then switch back to paren mode, and it (correctly) complains about unbalanced parens.

**Comment by shaunlebron** [comment]
Thanks for reporting, Colin.  will look into this

**Comment by cursive-ghost** [comment]
This is a previewCursorScope issue. I haven't had time to debug it yet, but entering:

```
(println \))
```

into the editor and then moving the caret around seems to alternately add or remove the caret every time parinfer is run.

**Comment by shaunlebron** [comment]
Thanks, I published 1.7.1 to fix this.  I poked around in the [demo editor](http://shaunlebron.github.io/parinfer/demo) and it seems to behave as expected.
